.github/actions/compare-lockfiles/compareLockfiles.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const core = require('@actions/core');
+const semverLt = require('semver/functions/lt');
+const { parseLockfile, getLockfilePackageVersions } = require('./helpers');
+
+const currentLockfile = parseLockfile(core.getInput('current-lockfile-path') || 'current');
```

Comment:
What happens if the `current-lockfile-path` is the same as `proposed-lockfile-path` how does this work?

Reply:
Great catch!